publishpress-checklists.php

```diff
@@ -86,3 +86,6 @@ function ($settings) {
     $plugin = new Plugin();
     $plugin->init();
 }
+
+// Ask for review
+require_once __DIR__ . '/modules/reviews/review-request.php';
```

Comment:
Considering it is a module, it should be registered in the function `LegacyPlugin::getModulesBasePath`. Just add the module slug to the `$defaultDirs` var, like:

```php
$defaultDirs = [
            'settings'    => PPCH_MODULES_PATH,
            'checklists'  => PPCH_MODULES_PATH,
            'permalinks'  => PPCH_MODULES_PATH,
            'yoastseo'    => PPCH_MODULES_PATH,
            'permissions' => PPCH_MODULES_PATH,
            'review' => PPCH_MODULES_PATH,
        ];
```

This require will not be needed, and it will be compatible with the plugin's modules system.